data/json/npcs/missiondef.json

```diff
@@ -134,6 +135,29 @@
       "failure": "It was a lost cause anyways..."
     }
   },
+  {
+    "id": "MISSION_FREE_MERCHANTS_EVAC_5",
+    "type": "mission_definition",
+    "name": "Find 40 Canned Food",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 3,
+    "value": 400000,
+    "item": "can_food",
```

Comment:
This will require player to find 40 units of tin cans?

Reply:
The intent was to require the player to find 40 food items contained in tin cans. And testing it with cans of beans and cans of soup shows that they do in fact fulfill quest requirements. However I hadn't though of how it would perceive empty cans. Just tested it, and apparently 40 empty tin cans also count for quest resolution purposes. Bit of an oversight there.